### PR TITLE
Neha0812 profile branch

### PR DESCRIPTION
**Pull request by neha0812**
Submitting the updated profile information

### CODE REVIEW

File: content/authors/neha0812/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+# Display name
+title: <Neha Sharma>
```
Comment (ianabc) [inline content/authors/neha0812/index.md:3]: Can you remove the angle brackets around your name? e.g.
```
title: Neha Sharma
```

File: content/authors/neha0812/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+organizations:
+- name: University of Alberta
+  url: https://www.ualberta.ca/index.html
+
+# A list of your interests, add as many as you want
+interests:
+- Mathematical Finance
+- Stochastic Analysis
+- Machine Leaning
+-Data Science
```
Comment (ianabc) [inline content/authors/neha0812/index.md:25]: YAML is very sensitive to white space, can you add a space between the `-` and the word `Data` e.g.
```
- Data Science
```

**Comment by ianabc** [comment]
Oh and just one other little thing, can you change the filename of the photo, it needs to be `avatar.jpg` not `avator.jpg`.

File: content/authors/neha0812/index.md (hunk 1 above)
Comment (neha0812) [reply content/authors/neha0812/index.md]: I have updated the errors. Thanks for pointing them out .

File: content/authors/neha0812/index.md (hunk 2 above)
Comment (neha0812) [reply content/authors/neha0812/index.md]: I have changed the .jpg file from avator to avatar. Thanks

**Comment by ianabc** [comment]
I made a mistake during my zoom call and told people to use the filename `index.md` but it should have been `_index.md`. I added one commit to tidy that up.

This looks great, so I'm going to merge it right away, thank you!

**Pull request merged**